libs/core/src/patterns/card-pattern-01/card-pattern-01.component.ts

```diff
@@ -89,6 +100,30 @@ export class GdsCardPattern01 extends withSizeXProps(
   @property({ reflect: false, attribute: 'aspect-ratio' })
   ratio: 'landscape' | 'square' = 'landscape'
 
+  /** Category */
+  @property({ reflect: false })
+  category = ''
+
+  /** Date */
+  @property({ reflect: false })
+  date = ''
+
+  /**
+   * Date format to use for displaying the date
+   * Available formats: dateLong, dateLongWithWeekday, dateShort, dateShortWithWeekday,
+   * dateOnlyNumbers, timeShort, timeLong, dateTimeLong, dateTimeLongWithWeekday, dateTimeShort
+   * @default 'dateTimeLong'
```

Comment:
Avoid putting hard coding type information in doc comments, since it can already be inferred from Typescript, and it's easy to forget to update the hard-coded list.